app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationHeader/integrationHeader.jsx

```diff
@@ -110,9 +110,9 @@ export const IntegrationHeader = (props) => {
               <Button
                 onClick={onResetProjectIntegration}
                 variant="ghost"
-                dataAutomationId="resetToGlobalIntegrationsButton"
+                dataAutomationId="projectIntegrationReset"
```

Comment:
Please do not change `dataAutomationId` without discussing with the team, as this change may affect the automated E2E tests.

Reply:
ok, thanks for mentioning that,
changed the value to initial.